### PR TITLE
Add log redaction info into clinic doc

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: clinic/clinic-user-guide-for-tiup.md [hunk 1]
```diff
@@ -72,6 +72,8 @@ summary: 详细介绍在使用 TiUP 部署的集群上如何通过 PingCAP Clini
     ```bash
     tiup diag config clinic.token ${token-value}
     ```
+3. [可选操作] 开启日志脱敏配置：
```
Comment (Oreoxmt) [inline clinic/clinic-user-guide-for-tiup.md:75]:
```suggestion

3. [可选操作] 开启日志脱敏配置：

```

File: clinic/quick-start-with-clinic.md [hunk 2]
```diff
@@ -50,9 +50,12 @@ PingCAP Clinic 目前处于 Technical Preview 阶段。
     {{< copyable "shell-regular" >}}
 
     ```bash
-    tiup diag config clinic.token ${token-value}
+    tiup diag config clinic.token ${token-value}    
     ```
 
+6. [可选操作] 开启日志脱敏配置：
```
Comment (Oreoxmt) [inline clinic/quick-start-with-clinic.md:56]:
```suggestion
6. [可选操作] 开启日志脱敏配置：

```

File: clinic/clinic-user-guide-for-tiup.md [hunk 3]
```diff
@@ -72,6 +72,8 @@ summary: 详细介绍在使用 TiUP 部署的集群上如何通过 PingCAP Clini
...
+    TiDB 在提供详细的日志信息时，可能会把数据库敏感的数据（例如用户数据）打印出来，如果用户不希望在本地日志以及上传到 Clinic 中的日志中包含敏感信息，可以开启日志脱敏配置。 具体操作请参考[日志脱敏](/log-redaction.md)。
```
Comment (Oreoxmt) [inline clinic/clinic-user-guide-for-tiup.md:76]:
```suggestion
    TiDB 在提供详细的日志信息时，可能会把数据库敏感的数据（例如用户数据）打印出来，如果用户不希望在本地日志以及上传到 Clinic 中的日志中包含敏感信息，可以开启日志脱敏配置。具体操作请参考[日志脱敏](/log-redaction.md)。
```

File: clinic/quick-start-with-clinic.md [hunk 4]
```diff
@@ -50,9 +50,12 @@ PingCAP Clinic 目前处于 Technical Preview 阶段。
     {{< copyable "shell-regular" >}}
 
     ```bash
-    tiup diag config clinic.token ${token-value}
+    tiup diag config clinic.token ${token-value}
```
Comment (Oreoxmt) [inline clinic/quick-start-with-clinic.md:53]:
```suggestion
    tiup diag config clinic.token ${token-value}
```

File: clinic/quick-start-with-clinic.md [hunk 5]
```diff
@@ -50,9 +50,12 @@ PingCAP Clinic 目前处于 Technical Preview 阶段。
     {{< copyable "shell-regular" >}}
 
     ```bash
-    tiup diag config clinic.token ${token-value}
+    tiup diag config clinic.token ${token-value}    
     ```
 
+6. [可选操作] 开启日志脱敏配置：
+    TiDB 在提供详细的日志信息时，可能会把数据库敏感的数据（例如用户数据）打印出来，如果用户不希望在本地日志以及上传到 Clinic 中的日志中包含敏感信息，可以开启日志脱敏配置。 具体操作请参考[日志脱敏](/log-redaction.md)。
```
Comment (Oreoxmt) [inline clinic/quick-start-with-clinic.md:57]:
```suggestion
    TiDB 在提供详细的日志信息时，可能会把数据库敏感的数据（例如用户数据）打印出来，如果用户不希望在本地日志以及上传到 Clinic 中的日志中包含敏感信息，可以开启日志脱敏配置。具体操作请参考[日志脱敏](/log-redaction.md)。
```

File: clinic/clinic-user-guide-for-tiup.md (hunk 3 above)
Comment (Oreoxmt) [inline clinic/clinic-user-guide-for-tiup.md:76]: @qqqdan 请问用户进行[日志脱敏](/log-redaction.md)配置时参考的内容是否为 TiDB 组件日志脱敏的部分呢？即 /log-redaction.md#tidb-组件日志脱敏
```suggestion
    TiDB 在提供详细的日志信息时，可能会把数据库敏感的数据（例如用户数据）打印出来，如果用户不希望在本地日志以及上传到 Clinic 中的日志中包含敏感信息，可以开启日志脱敏配置。 具体操作请参考[日志脱敏](/log-redaction.md#tidb-组件日志脱敏)。
```

**Review by lilyjazz — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: clinic/clinic-user-guide-for-tiup.md [hunk 6]
```diff
@@ -73,6 +73,10 @@ summary: 详细介绍在使用 TiUP 部署的集群上如何通过 PingCAP Clini
     tiup diag config clinic.token ${token-value}
     ```
 
+3. [可选操作] 开启日志脱敏配置：
```
Comment (en-jin19) [inline clinic/clinic-user-guide-for-tiup.md:76]:
```suggestion
3. 开启日志脱敏配置（可选步骤）。
```

File: clinic/clinic-user-guide-for-tiup.md [hunk 7]
```diff
@@ -73,6 +73,10 @@ summary: 详细介绍在使用 TiUP 部署的集群上如何通过 PingCAP Clini
...
+
+    TiDB 在提供详细的日志信息时，可能会把数据库敏感的数据（例如用户数据）打印出来，如果用户不希望在本地日志以及上传到 Clinic 中的日志中包含敏感信息，可以开启日志脱敏配置。具体操作请参考[日志脱敏](/log-redaction.md#tidb-组件日志脱敏)。
```
Comment (en-jin19) [inline clinic/clinic-user-guide-for-tiup.md:78]:
```suggestion
    TiDB 在提供详细的日志信息时，可能打印数据库的敏感信息（例如用户数据）。如果希望本地日志和上传到 Clinic Server 的日志里**不包含**敏感信息，你可以开启日志脱敏配置。具体操作请参考[日志脱敏](/log-redaction.md#tidb-组件日志脱敏)。
```

Comment (en-jin19) [inline clinic/clinic-user-guide-for-tiup.md:78]:
```suggestion
3. 开启日志脱敏配置（可选步骤）。

    TiDB 在提供详细的日志信息时可能会打印数据库的敏感信息（例如用户数据）。如果希望本地日志及上传到 Clinic Server 的日志中不带有敏感信息，你可以开启日志脱敏配置。具体操作请参考[日志脱敏](/log-redaction.md#tidb-组件日志脱敏)。
```

File: clinic/quick-start-with-clinic.md [hunk 8]
```diff
@@ -53,6 +53,10 @@ PingCAP Clinic 目前处于 Technical Preview 阶段。
     tiup diag config clinic.token ${token-value}
     ```
 
+6. [可选操作] 开启日志脱敏配置：
+
+    TiDB 在提供详细的日志信息时，可能会把数据库敏感的数据（例如用户数据）打印出来，如果用户不希望在本地日志以及上传到 Clinic 中的日志中包含敏感信息，可以开启日志脱敏配置。具体操作请参考[日志脱敏](/log-redaction.md#tidb-组件日志脱敏)。
```
Comment (en-jin19) [inline clinic/quick-start-with-clinic.md:58]:
```suggestion
6. 开启日志脱敏配置（可选步骤）。

    TiDB 在提供详细的日志信息时可能会打印数据库的敏感信息（例如用户数据）。如果希望本地日志及上传到 Clinic Server 的日志中不带有敏感信息，你可以开启日志脱敏配置。具体操作请参考[日志脱敏](/log-redaction.md#tidb-组件日志脱敏)。
```

**Review by en-jin19 — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**